build_tools/github_action/runner/linux/rocm_install.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# ROCm install
+wget https://repo.radeon.com/amdgpu-install/6.4/ubuntu/noble/amdgpu-install_6.4.60400-1_all.deb
```

Comment:
Marking "reviewed" until we resolve the rocm install questions. We _could_ punt on that if the existing runners are already doing this and this is just checking in the configuration.